README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img alt="" src="https://i.imgur.com/prUNzrf.png">
+  <img alt="pinia-plugin-persistedstate logo" src="https://i.imgur.com/prUNzrf.png">
```

Comment:
since this image is purely decorative, it should have an empty alt to be ignored by screen readers.

Reply:
`alt=""` is equivalent to `alt` in the resulting markup (idk why).
Setting this will also help when the image can't be loaded

Reply:
Ye, since it's purely decorative, its fine not having it load. I may also have it in the repo once i made the docs.
See this https://www.w3.org/WAI/tutorials/images/decorative/